pkg/common/DnnImageFeaturizer.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
(nit) you can drop `DefaultTargets="Build"` it doesn't make sense here. (and doesn't do anything)